chain/actions/src/redeem.rs

```diff
@@ -117,33 +122,37 @@ where
         Ok(receivers)
     }
 
-    #[tracing::instrument(level = "debug", skip(self))]
+    #[tracing::instrument(level = "debug", skip(self, min_value, only_aggregated))]
```

Comment:
Why would u not be interested in the arguments as well?

Reply:
I removed it by mistake while testing.